docs/encoders.md

```diff
@@ -39,3 +43,112 @@ That's it, now on every reboot the motor will turn in one direction until it fin
 
 * If you wish to scan for the index pulse in the other direction (if for example your axis usually starts close to a hard-stop), you can set a negative value in `<axis>.encoder.config.idx_search_speed`.
 * If your motor has problems reaching the index location due to the mechanical load, you can increase `<axis>.motor.config.calibration_current`.
+
+### Startup sequence notes
+
+The following are variables that MUST be set up for your encoder configuration. Your values will vary depending on your encoder:
+
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.mode = ENCODER_MODE_INCREMENTAL`
+
+The following are examples of values that MAY impact the success of calibration. Your values will vary depending on your setup:
+* `<axis>.motor.config.motor_type = MOTOR_TYPE_HIGH_CURRENT`
+* `<axis>.encoder.config.cpr = 4000`
+* `<axis>.encoder.config.calib_range = 0.05`
+* `<axis>.motor.config.calibration_current = 10.0`
+* `<axis>.motor.config.resistance_calib_max_voltage = 12.0`
```

Comment:
I'm not sure if the axis.motor stuff is relevant in this doc, maybe just have them run `AXIS_STATE_MOTOR_CALIBRATION` and go debug their motor issues somewhere else?

Reply:
Respectfully pushing back on this one. I know for the motor I used I do have to change these in order for it to go into calibration, and stated in the doc, "The following are examples of values that MAY impact the success of calibration."